.github/workflows/run-test.yml

```diff
@@ -0,0 +1,12 @@
+name: run-tests
```

Comment:
Great job on the initial set up

Reply:
Stevie is a wizz. Way to get this moving so quickly